appinventor/appengine/src/com/google/appinventor/client/explorer/folder/FolderManager.java

```diff
@@ -5,14 +5,13 @@
 
 package com.google.appinventor.client.explorer.folder;
 
+import com.google.gwt.json.client.JSONObject;
```

Comment:
The imports are not in lexicographical order as required by the Google Java Style.